docs/docs/introduction.mdx

```diff
@@ -0,0 +1,32 @@
+---
+route: /
+title: Introduction
+
+seoTitle: "Grafana Tanka: Define. Reuse. Override."
+description: >
+  Grafana Tanka is the robust configuration utility for your Kubernetes cluster, powered by the Jsonnet language.
+---
+
+import { Highlights } from "./home"
+import { Catcher } from "../src/components/home"
+
+<Catcher />
+
+## Highlights
+
+<Highlights
+  elems={{
+    Clean:
+      "The Jsonnet language expresses your Kubernetes apps more obviously than YAML ever did",
+    Reusable:
+      "Build application libraries, import them anywhere and even share them on GitHub!",
+    Concise:
+      "Using the Kubernetes library, you will never see boilerplate again!",
```

Comment:
```suggestion
      "Using the [Kubernetes library](https://jsonnet-libs.github.io/k8s-alpha/), you will never see boilerplate again!",
```
Not sure if markdown is interpreted here 🤔

Reply:
It's not. Will re-do highlights soon in a way that markdown works again